### PR TITLE
Add edit view for multiple locations page

### PR DESCRIPTION
**Pull request by avinhurry**
### Context

Implmenting the show page which comes after the `/new` page where the CSV is pasted. The number of rows will determine the number of locations. 

### Changes proposed in this pull request

- Add show views and hook up from the continue button on the previous page (`/new`)
- Add functionality to dynamically interpret how many location forms there will be and which one the user is on
- Add tests

### Guidance to review

Compre what has been done in this PR to the trello card and prototype.

### Screenshots

image

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased main
- [ ] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

**Comment by avinhurry** [comment]
[Bulk Uploads - CSV Flow](https://trello.com/c/DgfeqeZ3/1107-bulk-uploads-csv-flow)

**Comment by darokel** [comment]
Looking good! I noticed one quirk after playing around with the review app. After pasting in the CSV data and stepping through the locations for - when I return back to the csv upload page it still retains the previous input data:

Screenshot 2023-03-10 at 11 51 28

I think we want to clear this after you've done it

File: spec/forms/support/raw_csv_schools_form_spec.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+module Support
+  describe RawCSVSchoolsForm, type: :model do
+    subject { described_class.new(provider, params:) }
+
+    let(:provider) { create(:provider) }
+    let(:params) { { school_details: 'y,r,u,a,pain' } }
```
Comment (darokel) [inline spec/forms/support/raw_csv_schools_form_spec.rb:10]: 🤣

File: app/controllers/support/providers/locations/new_multiple_controller.rb [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+            end
+          else
+            max
+            render(:show)
+          end
+        end
+
+        private
+
+        def parased_csv_school_form
```
[the remark this answers is not in the document]
Comment (defong) [reply app/controllers/support/providers/locations/new_multiple_controller.rb]: Done

**Comment by defong** [comment]
> Looking good! I noticed one quirk after playing around with the review app. After pasting in the CSV data and stepping through the locations - when I return back to the csv upload page it still retains the previous input data:
> 
> Screenshot 2023-03-10 at 11 51 28
> 
> I think we want to clear this after you've done it

Can we leave that for the confirmation page as we need to retain the data for the back links until then?

**Review by darokel — APPROVED** [review]

**Pull request merged**